zcash_client_backend/src/wallet.rs

```diff
@@ -170,26 +170,26 @@ impl WalletSaplingSpend {
 /// `()` for sent notes.
 ///
 /// [`OutputDescription`]: sapling::bundle::OutputDescription
-pub struct WalletSaplingOutput<N, S> {
+pub struct WalletSaplingOutput<N, S, A> {
     index: usize,
     cmu: sapling::note::ExtractedNoteCommitment,
     ephemeral_key: EphemeralKeyBytes,
-    account: AccountId,
+    account: A,
     note: sapling::Note,
     is_change: bool,
     note_commitment_tree_position: Position,
     nf: N,
     recipient_key_scope: S,
 }
 
-impl<N, S> WalletSaplingOutput<N, S> {
+impl<N, S, A: Copy> WalletSaplingOutput<N, S, A> {
```

Comment:
Same comment here, prefer `Clone` as `Copy` may be more restrictive.

Reply:
`Clone` without `Copy` makes the `account()` function fail to compile.

Reply:
It would require an explicit call to `.clone()` - the `Copy` results in an _implicit_ copy, but more types can implement `Clone` than can implement `Copy`.

Reply:
This was fixable.